views/amp-access/article.html

```diff
@@ -78,27 +79,38 @@
     </header>
     <main role="main">
     <article>
-      <!-- 
-        This section is only visible by subscribers. The subscriber variable is 
-        defined in the authentification response data.
-      -->
-      <section amp-access="access AND subscriber" amp-access-hide class="meter-section" data-z-depth="2">
-        Thanks for being a subscriber. You rock!
-      </section>
+      <amp-user-notification
```

Comment:
So, this is actually a bit problematic at this time. We don't have a good integration with amp-user-notifications yet. The biggest issue is that once dismissed, the notification will not come back and there are other inconsistencies. If we need it only for "dismiss" - it's easy enough to simply put a button inside of the metered section with `on:hide` action:

```
<a on="tap:meter-notif.hide">[Close]</a>
```